doc/source/tune/examples/ml-frameworks.rst

```diff
@@ -103,10 +103,3 @@ At the end of these guides you will often find links to even more examples.
 
             End-to-end Example for Tuning a TensorFlow Model
 
-    .. grid-item-card::
```

Comment:
From our offline discussion, we should keep this example but just remove the "training" tag in the example library? (but it seems that there is no `training` tag originally)

So I think we should keep it in the Tune example page.

Reply:
Ah yes. Good catch. I forgot to put this back in. Will you follow up with moving the example to the `tune/examples` directory, @woshiyyya ?

Reply:
Just pushed a commit to add this back in.

Reply:
Ok I'll post a PR to move it to `tune/examples`.